### PR TITLE
Open Colors, add author, update deps

### LINKED ISSUE #25: Add author list
Maybe a small marldown file for every possible author with a few words about him/her.

### PR DESCRIPTION
**Pull request by lipis**
### https://excalidraw-blog-git-colors.excalidraw.now.sh/open-colors/

- [x] Added `author` and `link` to blog posts (Fixes #25)
- [x] Updated bunch of deps
- [x] Use `gatsby-embedder-excalidraw`
  - [x] Enable `gatsby-embedder-excalidraw` 
- [x] Mobile view Excalidraw preview overflows

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -62,9 +62,9 @@ module.exports = {
           "gatsby-remark-smartypants",
           {
             resolve: "gatsby-remark-embedder",
-            options: {
-              customTransformers: [require("./src/excalidraw-embed")],
-            },
+            // options: {
+            //   customTransformers: [require("gatsby-embedder-excalidraw")],
+            // },
```
Comment (lipis) [inline gatsby-config.js:67]: The plugin is not working :/

**Comment by lipis** [comment]
Something is wrong with the plugin: https://github.com/excalidraw/gatsby-embedder-excalidraw/pull/29

File: content/blog/open-colors/index.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+title: Open Colors
+date: 2020-10-29
+author: Lipis
+link: https://twitter.com/lipis
+---
+
+The biggest attribute of Excalidraw it's the simplicity. So even though we have the option to insert any color (using the [HEX](https://www.google.com/search?q=%23c0ffee&hl=en) value) we have just the _right amount_ (15) of the default colors, in three different shades.
```
Comment (dwelle) [inline content/blog/open-colors/index.md:8]:
```suggestion
One of the qualities of Excalidraw is its simplicity. Even though we have the option to use any color of the [spectrum](https://www.google.com/search?q=%23c0ffee&hl=en), we have decided to limit our palette to a curated set of 15 colors in three different shades.
```

File: content/blog/open-colors/index.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+<!-- end -->
+
+![Color pickers](color-pickers.png)
+
+I remember it was my very first [pull request](https://github.com/excalidraw/excalidraw/pull/378) to Excalidraw to add the [Open Colors](https://yeun.github.io/open-color/) open source color scheme and it was accepted the same day! I was super excited to say the least, because I knew this color scheme for a while now, but never found a great project to actually use it. This was it.
```
Comment (dwelle) [inline content/blog/open-colors/index.md:14]:
```suggestion
I remember it was my very first [pull request](https://github.com/excalidraw/excalidraw/pull/378) to Excalidraw to add the [Open Colors](https://yeun.github.io/open-color/) color scheme. I've known about this color scheme for a while now, but never found a great project to actually use it on. Excalidraw was a perfect fit.
```

File: content/blog/open-colors/index.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+They say a picture is worth a thousand words! I don't know what makes this color scheme so great, but what I know is that anything in Excalidraw is making it even better. (Click on the image below to open it in Excalidraw.)
```
Comment (dwelle) [inline content/blog/open-colors/index.md:16]:
```suggestion
They say a picture is worth a thousand words. I don't know what makes this color scheme so great, but something about Excalidraw is making it look even better. (Click on the image below to open it in Excalidraw.)
```

File: content/blog/open-colors/index.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+https://excalidraw.com/#json=5120999011909632,Y57VloPaA1LSKT4-1NTgNA
+
+Eventually we loved these colors so much, that we installed their [npm package](https://www.npmjs.com/package/open-color) and we are now using it everywhere. If you spot a color that is not an open color, sumbit a PR :)
```
Comment (dwelle) [inline content/blog/open-colors/index.md:20]:
```suggestion
Eventually we decided to install their [npm package](https://www.npmjs.com/package/open-color) so we don't have to hand-code every value. We use it both in [CSS](https://github.com/excalidraw/excalidraw/blob/master/src/css/_variables.scss), and [TypeScript](https://github.com/excalidraw/excalidraw/blob/master/src/colors.ts).
```

File: content/blog/open-colors/index.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+Here is how we are actually using them in [colors.ts](https://github.com/excalidraw/excalidraw/blob/master/src/colors.ts) in our codebase.
```
Comment (dwelle) [inline content/blog/open-colors/index.md:22]:
```suggestion
```

File: content/blog/open-colors/index.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+Open Colors are coming with MIT License, which means you can use it anywhere you like without restrictions.
```
Comment (dwelle) [inline content/blog/open-colors/index.md:24]:
```suggestion
Open Colors come with MIT License, which means you can use it anywhere you like without restrictions.
```

File: content/blog/open-colors/index.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+---
+
+**P.S.** We owe a huge thanks to the [Open Colors](https://yeun.github.io/open-color/) creator, as we trully believe that part of Exaclidraw's success is these colors as well :)
```
Comment (dwelle) [inline content/blog/open-colors/index.md:28]: Thanks [@yeun](https://github.com/yeun) and the contributors for creating and maintaining the project!

**Comment by dwelle** [comment]
Made some copy editing. Btw, shouldn't the image be below the `picture is worth a thousand words` paragraph, and be wrapped in the excal URL as you mention?

**Comment by dwelle** [comment]
I removed the solicitation for PR because I bet people would start putting OC into tests and stuff. We also hard-code some strings like in element default values etc., where I think it's fine to not use OC. We also not use OC deliberately e.g. in renderer when editing a line, because I couldn't find an OC red color shade that looked good there.

**Review by topless — APPROVED** [review]

File: content/blog/open-colors/index.md [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+date: 2020-11-09
+author: Lipis
+link: https://twitter.com/lipis
+---
+
+One of the qualities of Excalidraw is its simplicity. Even though we have the option to use any color of the [spectrum](https://www.google.com/search?q=%23c0ffee&hl=en), we have decided to limit our palette to a curated set of 15 colors in three different shades.
+
+<!-- end -->
+
+The [Open Colors](https://yeun.github.io/open-color/) color scheme has a total of 13 colors with 10 different brightnesses (0-9). For the canvas background we chose the lightest of the values (0), for the strokes the darkest one (9) and for the background fill of the elements slightly lighter than the stroke, we went with the 6th (or the 7th, dependas on how you count them).
```
Comment (dwelle) [inline content/blog/open-colors/index.md:12]:
```suggestion
The [Open Colors](https://yeun.github.io/open-color/) color scheme has a total of 13 colors with 10 different brightnesses (0-9). For the canvas background we chose the lightest of the values (0), for the strokes the darkest one (9), and for the element background fill we went with the 6th (or the 7th, depending on how you count them).
```

Comment (dwelle) [inline content/blog/open-colors/index.md:12]:
```suggestion
The [Open Colors](https://yeun.github.io/open-color/) color scheme has a total of 13 colors with 10 different brightness levels (0-9). For the canvas background we chose the lightest of the values (0), for the strokes the darkest one (9) and for the background fill of the elements slightly lighter than the stroke, we went with the 6th (or the 7th, dependas on how you count them).
```

File: content/blog/open-colors/index.md [hunk 10]
```diff
@@ -0,0 +1,26 @@
+---
+title: Open Colors
+date: 2020-11-09
+author: Lipis
+link: https://twitter.com/lipis
+---
+
```
Comment (dwelle) [inline content/blog/open-colors/index.md:7]:
```suggestion

(Part of a series where we pick apart various libraries and projects we use on Excalidraw.)
```

Comment (dwelle) [reply content/blog/open-colors/index.md:7]: Best if we also come up with a title prefix as discussed.

Comment (lipis) [reply content/blog/open-colors/index.md:7]: I added that as a note in frontmatter...

**Pull request merged**